docs/UserGuide.md

```diff
@@ -116,17 +116,17 @@ This detailed guide will walk you through how to make the most of **EduManage**
-|  10   |   Task Deadline   |               `d/TASK_DEADLINE`                | Should be in the format `YYYY-MM-DD`, and should only start from today's date (according to your computer's local time).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-|  11   | Task Description  |              `t/TASK_DESCRIPTION`              | Can take any values except prefixes already reserved, and should not be blank.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+|  10   |   Task Deadline   |               `d/TASK_DEADLINE`                | Should be in the format `YYYY-MM-DD`. While any date can be chosen, it is recommended to stick to recent dates.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|  11   | Task Description  |              `t/TASK_DESCRIPTION`              | Can take any values, but should not be blank. Note that using certain prefixes may cause an error.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Same as above, change to "Note that using reserved prefixes within the task description may cause an error." for clarity?